ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-release-4.10.yaml

```diff
@@ -95,18 +95,6 @@ tests:
         installed on the cluster shouldn't report any alerts in firing state apart
         from Watchdog and AlertmanagerReceiversNotConfigured
     workflow: openshift-e2e-openstack-ccm
-- as: e2e-openstack-ccm-install
```

Comment:
This seemed duplicate of the above job as it also create a featuregate. So, I removed it.

Reply:
Are the differences not that one bootstraps as techpreview and the other bootstraps and then changes over to tech preview once bootstrapped? So they test slightly different configurations